shopping.html

```diff
@@ -66,6 +66,20 @@ <h1 class="shopHeader">Shopping with Pina Cullotte-a</h1> <!-- CUSTOMIZE THIS FO
     <!-- TODO Div for the Product Details <<Also this div is hide>>
          it show only if someone click on the product name -->
 
+         <section>
+          <div class="container">
+              <div class="card mb-3" style="max-width: 540px;">
+                  <div class="row no-gutters justify-content-md-center">
```

Comment:
Is there supposed to be content in this section?  When I view the shopping page, all I see is a blank line that's off-center and doesn't seem like it belongs.

![Screen Shot 2020-05-16 at 9 03 40 AM](https://user-images.githubusercontent.com/10491407/82121655-53f78980-9754-11ea-9211-f65e10d7014f.png)